packages/api/src/clickhouse/index.ts

```diff
@@ -1096,14 +1097,39 @@ const buildEventSeriesQuery = async ({
       ? buildSearchColumnName(propertyTypeMappingsModel.get(field), field)
       : '';
 
-  const hasGroupBy = groupBy != '' && groupBy != null;
   const isCountFn = aggFn === AggFn.Count;
-  const groupByField =
-    hasGroupBy &&
-    buildSearchColumnName(propertyTypeMappingsModel.get(groupBy), groupBy);
+  const groupByFields: string[] = [];
```

Comment:
mega nit: probably call this groupByColumnNames, fields to me usually means the user-facing field I think